docs/source/changes.rst

```diff
@@ -8,6 +8,12 @@ Bugfixes:
 
 - ``unitary_tools.lifted_gate()`` was not properly handling modifiers such as ``DAGGER`` and ``CONTROLLED``.
 
+Improvements and Changes:
+
+- The test suite for `pyquil.operator_estimation` now has an (optional) faster version
```

Comment:
let's just make sure it's clear that this is a pytest flag. i would do it but i cant push to your branch

```suggestion
- The test suite for `pyquil.operator_estimation` now has an (optional) faster version
  that uses fixed random seeds instead of averaging over several experiments. This can
  be enabled with the `--use-seed` command line input when running `pytest` (gh-928).
```